<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectProviderDialog</class>
 <widget class="QDialog" name="ConnectProviderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>567</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connect Provider Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_5">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_4">
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
     <property name="bottomMargin">
      <number>9</number>
     </property>
     <item row="0" column="0" colspan="3">
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="provider_label_placeholder">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>provider_placeholder</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QFrame" name="frame">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Get your credentials: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="help_link_placeholder">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;http://dummy.com&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;help_link_placeholder&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
              <property name="openExternalLinks">
               <bool>true</bool>
              </property>
              <property name="textInteractionFlags">
               <set>Qt::TextBrowserInteraction</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="place_here_your_placeholder">
           <property name="text">
            <string>paste_here_your_placeholder:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QTextEdit" name="credentials_textedit"/>
       </item>
       <item row="2" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="save_config_button">
           <property name="text">
            <string>Save Credentials</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="ping_connection_button">
           <property name="text">
            <string>Test Connection</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="1">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QPushButton" name="load_voices_button">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>Load Voices</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QPushButton" name="close_button">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QPushButton" name="confirm_button">
         <property name="text">
          <string>Confirm</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QVBoxLayout" name="status_bar_layout">
     <property name="spacing">
      <number>0</number>
     </property>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>close_button</sender>
   <signal>clicked()</signal>
   <receiver>ConnectProviderDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>354</x>
     <y>625</y>
    </hint>
    <hint type="destinationlabel">
     <x>126</x>
     <y>492</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>save_config_button</sender>
   <signal>clicked()</signal>
   <receiver>ConnectProviderDialog</receiver>
   <slot>saveCredentials()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>233</x>
     <y>564</y>
    </hint>
    <hint type="destinationlabel">
     <x>367</x>
     <y>419</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ping_connection_button</sender>
   <signal>clicked()</signal>
   <receiver>ConnectProviderDialog</receiver>
   <slot>testConnection()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>117</x>
     <y>564</y>
    </hint>
    <hint type="destinationlabel">
     <x>138</x>
     <y>461</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>load_voices_button</sender>
   <signal>clicked()</signal>
   <receiver>ConnectProviderDialog</receiver>
   <slot>loadVoices()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>352</x>
     <y>592</y>
    </hint>
    <hint type="destinationlabel">
     <x>432</x>
     <y>468</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>confirm_button</sender>
   <signal>clicked()</signal>
   <receiver>ConnectProviderDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>624</y>
    </hint>
    <hint type="destinationlabel">
     <x>108</x>
     <y>615</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>saveCredentials()</slot>
  <slot>testConnection()</slot>
  <slot>loadVoices()</slot>
 </slots>
</ui>
